packages/iTwinUI-react/package.json

```diff
@@ -29,10 +29,11 @@
     "ux"
   ],
   "scripts": {
-    "build": "yarn clean && yarn copy-files && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json",
-    "build:watch": "yarn clean && yarn copy-files && concurrently \"tsc -p tsconfig.cjs.json --watch\" \"tsc -p tsconfig.esm.json --watch\"",
-    "clean": "yarn clean:coverage && rimraf esm && rimraf cjs",
+    "build": "yarn clean:build && yarn copy-files && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json",
+    "build:watch": "yarn clean:build && yarn copy-files && concurrently \"tsc -p tsconfig.cjs.json --watch\" \"tsc -p tsconfig.esm.json --watch\"",
+    "clean:build": "yarn clean:coverage && rimraf esm && rimraf cjs",
     "clean:coverage": "rimraf coverage",
+    "clean": "yarn clean:coverage && yarn clean:build && rimraf node_modules",
```

Comment:
Maybe also clean the `.turbo` folder in every package to remove the cache

Reply:
done